wkcuber/convert_nifti.py

```diff
@@ -102,6 +102,9 @@ def to_target_datatype(
     else:
         factor = np.iinfo(data.dtype).max / np.iinfo(target_dtype).max
 
+    if factor == 0:
+        factor = 1
```

Comment:
I think, this can only happen when `data.max() == 0`, right? (at least with the current implementation)

I'd suggest to change the if-conditional accordingly, to make this fallback more strict. In general, dividing-by-zero is a bug and just doing "nothing" (i.e., dividing by 1) might not always be the correct way to deal with edge cases. If the implementation changes at some point and introduces a new divide-by-zero bug, the `if factor == 0` would just silently shadow it (which might be correct or wrong).

I'd suggest the following:
```
if data.max() == 0:
  logging.warning("Don't rescale data since maximum is 0")
  factor = 1
```

Do you agree?